client/landing/gutenboarding/components/domain-picker/suggestion-item-placeholder.tsx

```diff
@@ -6,9 +6,8 @@ import React, { FunctionComponent } from 'react';
 const DomainPickerSuggestionItemPlaceholder: FunctionComponent = () => {
 	return (
 		<div className="domain-picker__suggestion-item">
-			<div className="domain-picker__suggestion-item-name placeholder">
-				<input disabled className="domain-picker__suggestion-radio-button" type="radio" />
-			</div>
+			<input disabled className="domain-picker__suggestion-radio-button" type="radio" />
+			<div className="domain-picker__suggestion-item-name placeholder"></div>
```

Comment:
Nit: We generally prefer self-closing in cases like these.

```suggestion
			<div className="domain-picker__suggestion-item-name placeholder" />
```